README.md

```diff
@@ -1,143 +1,83 @@
   </a>
 </p>
 
-## Overview
+<br/>
+<br/>
 
-CodeLingo finds and fixes issues in existing code and ensures the same issues don't sneak in with new pull requests. CodeLingo also generates your contributor docs thus automating three cornerstones (bug fixing, reviewing and doc updates) to scale up your code quality.
+[![Dashboard](https://img.shields.io/badge/-Dashboard-informational?style=flat)](https://dash.codelingo.io)
+[![codelingo.io](https://img.shields.io/badge/-codelingo.io-yellowgreen?style=flat)](https://codelingo.io)
+[![Slack](https://img.shields.io/badge/Slack-codelingo--dev-blueviolet?style=flat)](https://codelingo.slack.com/messages/codelingo-dev)
+[![VSCode Extension](https://img.shields.io/badge/VSCode-Capture-Blue?style=flat)](https://marketplace.visualstudio.com/items?itemName=codelingo.codelingo)
+[![Docs](https://img.shields.io/badge/Docs-CodeLingo-green?style=flat)](https://docs.codelingo.io)
+[![Playground](https://img.shields.io/badge/Playground-CodeLingo-orange?style=flat)](https://playground.codelingo.io)
+[![Twitter](https://img.shields.io/twitter/follow/codelingoapp?label=Follow%20Us)](https://twitter.com/codelingoapp)
 
 
-<!-- TODO CLQL tutorial -->
+CodeLingo learns your coding rules to auto identify issues and teach your best practices; keeping you nimble on a fast-moving codebase.
+Use simple in-line workflows to document your best practices and enforce them in real-time.
 
-## @CodeLingoBot Quick Start
+# Installation
 
-@CodeLingoBot fixes bugs, automates code reviews and updates contributor docs for repos on GitHub based on rules, called Rules, defined in codelingo.yaml files. There are several ways you can interact with @CodeLingoBot. The simplest is to trigger actions by commenting on a Pull Request:
+## GitHub
 
-+ `@CodeLingoBot review` reviews the PR based on default Rules for the repo's language or custom Rules added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@review`</summary>
-    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
-    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
-+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Rules for the repo's language or custom Rules added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@rewrite`</summary>
-  
-    + `@CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.
+To have CodeLingo automatically review every new Pull Request to your repo, [install CodeLingo on your repo](https://github.com/marketplace/codelingo) and add some rules to your repo via the [dashboard](https://dash.codelingo.io).
```

Comment:
is this where we want to focus? isn't it about living docs – capture rules and discuss them is what we can _actually_ do right now

Reply:
+1